.htaccess

```diff
@@ -7,7 +6,5 @@ RewriteRule ^redirect - [L,NC]
 
 #Rewrite any subdomains to the correct script whichs checks where to forward to
 RewriteCond %{HTTP_HOST} !www.tum.sexy [NC]
-RewriteCond %{HTTP_HOST} ^(www\.)?([a-z0-9-]+).tum.sexy [NC]
-RewriteRule ^(.*)$ /redirect/index.php?sub=%2 [L]
-
-
+RewriteCond %{HTTP_HOST} ^(?:www\.)?(?:([a-z0-9-]+)\.)?([a-z0-9-]+).tum.sexy [NC]
+RewriteRule ^(.*)$ /redirect/index.php?siteType=%3&redirectUrl=%1 [L]
```

Comment:
This needs to go to PHP. We should just redirect all requests with the `!-f` flag to the redirect script.

Reply:
Plus: we can test this regex then

Reply:
Plus you mixed up the parameters:
* `siteType` is it the moodle site or normal redirect
* `redirectURL` the usual part that we lookup in our table

Reply:
> This needs to go to PHP. We should just redirect all requests with the !-f flag to the redirect script.
I don't know how to do that 🙈